### PR TITLE
AVS-15

### PR DESCRIPTION
**Pull request by Andries16**

### CODE REVIEW

File: database/seeds/DatabaseSeeder.php [hunk 1]
```diff
@@ -4,13 +4,9 @@
...
 {
-    /**
-     * Seed the application's database.
-     *
-     * @return void
-     */
     public function run()
     {
          $this->call(UsersTableSeeder::class);
+         $this->call(CtegoriesTableSeeder::class);
```
Comment (arsdu24) [inline database/seeds/DatabaseSeeder.php]: Typo 
```suggestion
         $this->call(CategoriesTableSeeder::class);
```

File: database/seeds/CtegoriesTableSeeder.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class CtegoriesTableSeeder extends Seeder
```
Comment (arsdu24) [inline database/seeds/CtegoriesTableSeeder.php]: Typo 
```suggestion
class CategoriesTableSeeder extends Seeder
```

File: app/Image.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?php
```
Comment (arsdu24) [inline app/Image.php]: Why, do we use this model

**Review by arsdu24 — APPROVED** [review]

File: app/Image.php (hunk 3 above)
Comment (Andries16) [reply app/Image.php]: when I was doing the dropzone I had some problems and I tried with the model, but after it was received without it

**Pull request merged**